src/VisualStudio/Core/Test.Next/VisualStudioTest.Next.csproj

```diff
@@ -85,6 +84,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Validation" Version="$(MicrosoftVisualStudioValidationVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
     <PackageReference Include="StreamJsonRpc" Version="$(StreamJsonRpcVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowVersion)" />
```

Comment:
What package is actually shipping this binary? Does VS ship it and we've been relying on that copy? I didn't think it was part of the framework.

Reply:
yes, it's shipping with VS as version 4.5.24. You were relying on it indirectly by referencing Microsoft.Tpl.Dataflow package, which contains System.Threading.Tasks.Dataflow.dll.

Reply:
@olegtk I guess my question is "somebody other than us is responsible for making sure it gets on the user's machine?"